src/Day1.hs

```diff
@@ -91,11 +93,14 @@ updatePlayerPos Input{keys=keys} (x,y) = (x',y')
 tileMap :: State -> Map (Int, Int) Tile
 tileMap st = case scene st of
   Scene'Start -> text "DAY 1" White1 (screenW `div` 2 - 2, screenH `div` 2)
-  Scene'Play -> mergeTiles clear $ fromList [ ( (x,y), Tile Nothing Nothing (Just wh1) ) ]
+  Scene'Play -> mergeTiles playerTile ( mergeTiles testDoors ( mergeTiles testWalls clear ) )
   Scene'GameOver -> fromList []
   where
     (x,y) = playerPos (player (playState st))
     clear = clearTileMap (colorFromRoomIndex (playerRoom (player (playState st))))
+    playerTile = fromList [ ( (x,y), Tile Nothing Nothing (Just wh1) ) ]
+    testDoors = doorTileMap [1, 2, 3] [(9,8), (10, 13), (3, 7)]
```

Comment:
The data for doors and walls should be coming from somewhere in the `State`

Reply:
Okay will do. I will create a new Door datatype perhaps.

Reply:
If there are just list of indices they don't need a special type.

Reply:
Wait. Nevermind.